Add cross-cloud-region-hybrid + hardhat (#323)
chainstack/docs: docs/glossary/hybrid.md

---
meta:
  - name: description
    content: A hybrid network is the combination of nodes deployed in the Chainstack-provided cloud providers and private hosting.
  - name: keywords
    content: hybrid blockchain deployment on-prem cloud
---

# Hybrid

> Deployment of nodes of the same network in Chainstack-provided cloud hosting providers and private hosting.

Reviewer: Technically can be also a combination of multiple private hostings

Author: Fixed.

> Deployment of nodes of the same network in Chainstack-provided cloud hosting providers and private hosting, or a combination of private hostings.

Reviewer: I think can be simplified a bit

```suggestion
Deployment of nodes of the same network in any combination of Chainstack-provided cloud hosting providers and private hosting providers.
```
